optaplanner-docs/src/main/asciidoc/QuickStart/CloudBalancingDomainModel/CloudBalancingDomainModel-section.adoc

```diff
@@ -120,14 +125,14 @@ The `CloudBalance` class has a [path]_@PlanningSolution_
  annotation.
 
 * It holds a list of all computers and processes.
-* It represents both the planning problem and (if it's initialized) the planning solution.
+* It represents both the planning problem and (if it is initialized) the planning solution.
 
-. Planner needs to retrieve the collection of processes that it can change, therefore we annotate the getter `getProcessList()` with ``@PlanningEntityCollectionProperty``.
+. The class holds a collection of processes that Planner can change. We annotate the getter `getProcessList()` with ``@PlanningEntityCollectionProperty``, so that Planner can retrieve the processes that it can change. To save a solution, Planner initializes a new instance of the class with the list of changed processes.
```

Comment:
-1 It doesn't do that.

Reply:
Could you then explain what it does?

Reply:
It indeed retrieves the process that Planner can change. But it's not a class and the "to save ..." sentence doesn't make sense.

Reply:
Changed, will push soon.